core/ui/src/main/res/values/strings.xml

```diff
@@ -430,6 +430,11 @@
     <string name="formatPercent">%1$.0f%%</string>
     <string name="basal">Basal</string>
     <string name="basalpct">Basal %</string>
+    <string name="sensitivity">Sensitivity</string>
+    <string name="isf_for_carbs">ISF for carbs absorption: %1$.1f</string>
```

Comment:
How could we improve this string without making it to long? This ISF value is also used in Bolus calculator BG / Trend. This is very relevant information that we should present some how.

ISF for bolus and COB calculation
ISF for bolus calculator and COB calculation
ISF for bolus calculator and carbs absorption
ISF for bolus calc and carbs absorption

Reply:
Difficulty is none of your proposal fit in one line only for label + value (and they are in english language)
We should also think of translation (even if not directly under control), that can be longer...
All these label re in string so can be adjusted later if everyone agree on a better proposal ;-)